### ISSUE TITLE
Ability to change the Letterboxing color (padded rectangles)

### ISSUE THREAD

**Issue by aytunch** [brianegan/chewie#267]
This is for videos which we want to keep the original aspect ratio. Is there a way to customize the Letterbox color to be padded to the video instead of Black?

**Comment by andersmmg** [comment]
Pretty sure that's what the `placeholder` option on the `ChewieController` widget is for. I just use a `Container` with a color.

**Comment by realappie** [comment]
@andersmmg I am not sure if thats true, I added this widget as `placeholder`

```dart
 Container(
    color: Colors.red
 )
```

Yet I still have 'black' padding.

![](https://i.gyazo.com/bbdffdfe0fa9898ddc168197347ca533.gif)

I think we need to make the following line customisable on the `ChewieController`, for those with videos with odd aspect ratios.

https://github.com/brianegan/chewie/blob/db2489bce51b54ca7e73f803ab24f49d169d3483/lib/src/chewie_player.dart#L86

**Comment by andersmmg** [comment]
That's weird, I tried this morning and it worked exactly as I expected. Are you sure the letterbox isn't embedded in the video file or something?

**Comment by realappie** [comment]
@andersmmg Yes. I am sure, because the black padding changes position depending on width/height of video.
